src/tests/purchase.virtual-currencies.test.ts

```diff
@@ -7,34 +7,34 @@ import type { VirtualCurrencies } from "../entities/virtual-currencies";
 import { http, HttpResponse } from "msw";
 import { PurchasesError, ErrorCode } from "../entities/errors";
 
-describe("getVirtualCurrencies", () => {
-  const appUserIDWith3Currencies = "test-app-user-id-with-3-currencies";
-  const appUserIDWith0Currencies = "test-app-user-id-with-0-currencies";
+const appUserIDWith3Currencies = "test-app-user-id-with-3-currencies";
```

Comment:
The git diff is a little confusing here. 

All this does is move `appUserIDWith3Currencies`, `appUserIDWith0Currencies` and `createExpectedVirtualCurrenciesWith3Currencies()` out of the `describe("getVirtualCurrencies", () => {}` block so it can be shared across the different `describe` test contexts.